NuGet.Config

```diff
@@ -8,10 +8,19 @@
     <add key="myget-legacy@Local" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy%40Local/nuget/v3/index.json" />
     <add key="nuget-build" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json" />
     <add key="vside" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/msft_consumption%40Local/nuget/v3/index.json" />
+    <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
+    <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
```

Comment:
If we don't need the `Microsoft.VisualStudio.Shell.ViewManager` package, as suggested in [this comment](https://github.com/NuGet/NuGet.Client/pull/5920/files#r1706289017), then we probably don't need these sources. Both of these feeds are configured as upstream sources for the `msft_consumption` feed. The `Microsoft.VisualStudio.Markdown.Platform` package is already uploaded to the [upstream source](https://dev.azure.com/azure-public/vside/_artifacts/feed/msft_consumption/NuGet/Microsoft.VisualStudio.Markdown.Platform/upstreams) of the `msft_consumption` feed. Since we are using the `local` view, we can select the package version in the UI and click on `Save to Feed` to add the package to the local view of the consumption feed.